[2023-06-28T21:47:18.518+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data_new.get_s3_raw_data manual__2023-06-28T17:32:27-04:00 [queued]>
[2023-06-28T21:47:18.528+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data_new.get_s3_raw_data manual__2023-06-28T17:32:27-04:00 [queued]>
[2023-06-28T21:47:18.528+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T21:47:18.528+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-28T21:47:18.528+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T21:47:18.572+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): get_s3_raw_data> on 2023-06-28 21:32:27+00:00
[2023-06-28T21:47:18.578+0000] {standard_task_runner.py:55} INFO - Started process 4011 to run task
[2023-06-28T21:47:18.581+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data_new', 'get_s3_raw_data', 'manual__2023-06-28T17:32:27-04:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data_new.py', '--cfg-path', '/tmp/tmpzr6jawdy']
[2023-06-28T21:47:18.583+0000] {standard_task_runner.py:83} INFO - Job 295: Subtask get_s3_raw_data
[2023-06-28T21:47:18.693+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data_new.get_s3_raw_data manual__2023-06-28T17:32:27-04:00 [running]> on host d9b032dfc97f
[2023-06-28T21:47:19.105+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data_new
AIRFLOW_CTX_TASK_ID=get_s3_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2023-06-28T21:32:27+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-28T17:32:27-04:00
[2023-06-28T21:47:19.126+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-28T21:47:19.127+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-28T21:47:33.397+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-28T21:47:33.432+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data_new, task_id=get_s3_raw_data, execution_date=20230628T213227, start_date=20230628T214718, end_date=20230628T214733
[2023-06-28T21:47:33.528+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-28T21:47:33.564+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
